shopinvader_assortment/migrations/13.0.2.1.0/pre-migration.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 ForgeFlow <http://www.forgeflow.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+_column_renames = {
+    "shopinvader_backend": [("product_assortment_id", None),],
+}
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.rename_columns(env.cr, _column_renames)
```

Comment:
we should check if column exists before trying to apply the rename

Reply:
@JordiMForgeFlow done